### ISSUE TITLE
[Usability] User can't identify an element or its state.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80591]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User can't identify an element or its state.
**VA.gov Experience Standard - category:** Usability
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the "Ask for your claim decision" page (eg. test user Kyle Cole/+23, [/your-claims/600494815/ask-va-to-decide](https://staging.va.gov/track-claims/your-claims/600494815/ask-va-to-decide)), the submit button is disabled unless you check the "I have submitted all evidence..." checkbox.

Disabled elements pose challenges for screen reader users --- see the [design system guidance on buttons](https://design.va.gov/components/button/#do-not-disable-buttons) for a discussion of this. It also looks like this issue has already been flagged in #78853.

## Recommended action

The recommended way to handle this would be to validate on submit and have an error message on the checkbox.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by davidakennedy** [comment]
Noting here that the team can probably work from the existing ticket: #78853.

**Comment by jacobworrell** [comment]
this feedback has been incorporated into the designs for the new 5103 evidence request intitiative.

**Comment by pmclaren19** [comment]
This ticket will be blocked until we release the epic work for Improving 5103 tracked items.

**Comment by pmclaren19** [comment]
This has been resolved.
